examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6549,6 +6549,12 @@ endpoint 1 {
 
   server cluster Switch {
     emits event SwitchLatched;
+    emits event InitialPress;
```

Comment:
This is not a valid configuration for a Switch cluster.  That's why there were not enabled: you can have the LS feature or the MS feature, but not both.

Reply:
Acked, review MS feature